test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -292,6 +292,48 @@
           (testing (format "%s %s function works as expected on %s column for driver %s" op unit col-type driver/*driver*)
             (is (= (set expected) (set (test-datetime-math query))))))))))
 
+(defn- close? [t1 t2 period]
+  (and (t/before? (t/instant t1) (t/plus (t/instant t2) period))
+       (t/after? (t/instant t1) (t/minus (t/instant t2) period))))
+
+(deftest now-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :date-arithmetics)
+    (testing "should return the current time"
+      ;; Allow a 30 second window for the current time to account for any difference between the time in Clojure and the DB
+      (doseq [timezone [nil "America/Los_Angeles"]]
+        (mt/with-temporary-setting-values [report-timezone timezone]
+          (is (= true
+                 (-> (mt/run-mbql-query venues
```

Comment:
I think sometimes the `:where` clause goes through a different pathway? Maybe do a `select count(*) from products where created_at < now()` and just assert that it returns a count greater than zero?